tests/test_converters.py

```diff
@@ -106,7 +106,7 @@ def test_use_all(self, data):
         assert set(dataset.c.coords) == {"chain", "draw", "c1", "c2"}
 
     def test_missing_coords(self, data):
-        dataset = convert_to_dataset(data.datadict, coords=None, dims=data.dims)
+        dataset = convert_to_dataset(data.datadict, coords={}, dims=data.dims)
```

Comment:
This might be an error in the docstring. `None` should be valid and is probably the default value.

Reply:
i think I changed this by mistake, mypy is saying that it can be `None`.